packages/web/utils/cache.ts

```diff
@@ -15,28 +15,40 @@ const isTestEnv = process.env.NODE_ENV === "test";
  *  NOTE: Uses an LRUCache in test environment to avoid hitting the remote resource as Vercel KV createClient secret env vars are not available in GH action.
  *
  *  WARNING: Only available in dev/prod node runtime (not browser).
+ *
+ *  Falls back to in-memory cache if the remote client is not able to be created.
  */
 export class RemoteCache implements Cache {
   protected kvStore: VercelKV | null = null;
 
-  protected testEnvKvStore = isTestEnv
-    ? new LRUCache<string, CacheEntry>({ max: 50 })
-    : null;
+  protected fallbackCache: Cache | null = null;
 
   name = "RemoteCache";
 
   constructor() {
     if (!isTestEnv) {
-      this.kvStore = createClient({
-        url: process.env.TWITTER_KV_STORE_REST_API_URL!,
-        token: process.env.TWITTER_KV_STORE_REST_API_TOKEN!,
-      });
+      try {
+        this.kvStore = createClient({
+          url: process.env.TWITTER_KV_STORE_REST_API_URL!,
+          token: process.env.TWITTER_KV_STORE_REST_API_TOKEN!,
+        });
+      } catch (e) {
+        console.error(
+          "Failed to create RemoteCache client. Falling back to in-memory cache.."
+        );
+        this.fallbackCache = new LRUCache<string, CacheEntry>({ max: 50 });
```

Comment:
Thoughts on using the `DEFAULT_LRU_OPTIONS` here?

Reply:
I thought about that, but I wanted to avoid having web config be a dependency with this file.

Perhaps we can move that const to this file? Since this file manages cache utils?

Reply:
I agree. In the future, when we move `server` to a new package, we'll most likely need these variables.